config.json

```diff
@@ -555,6 +555,18 @@
         "loops",
         "math"
       ]
+    },
+    {
+      "slug": "circular-buffer",
+      "uuid": "92788dbc-feb2-43a6-99c0-af21ce318b01",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Let's change `unlocked_by` to "raindrops" instead of null. That will be a decent place for it until #237 is addressed.

Everything else should be the same.